docs/debugging_lnd.md

```diff
@@ -12,36 +12,193 @@ data ahead of time.
 
 ## Debug Logging
 
-You can enable debug logging in `lnd` by passing the `--debuglevel` flag. For
-example, to increase the log level from `info` to `debug`:
+LND supports different logging levels and you can also specify different logging
+levels per subsystem. This makes it easy to focus on a particular subsystem
+without clogging up the logs with a lot of noise. One can either set the logging
+in the lnd.conf file or pass the flag `--debuglevel` with the specified level 
+when starting lnd.
+
+LND supports the following logging levels (see [log.go](/build/log.go) and
+[sample-lnd.conf](/sample-lnd.conf) for more info):
+
```

Comment:
Non blocking: Would it be helpful to provide a definition of each log level type? A definition which can provide context on which level is appropriate when

Reply:
hmm I think these log levels are so universal in computer science that it should be self-explanatory ?